packages/odata-common/test/common-entity.ts

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable */
```

Comment:
I guess this is not needed any more as we have the generated one?

Reply:
I put both classes: entity and complex type in one file also the order of the attributes is not right. We disable the lint for all generated test cases because also the double export class and namespace is not ok. I did not want to disable the lint for any other file in odata-common so I used the disable.